os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c

```diff
@@ -1000,18 +1001,19 @@ u32 app_mpu_nocache_init(void)
 	mpu_region_cfg(mpu_entry, &mpu_cfg);
 #endif
 
-#if 0 //fix me!
 	/* set PSRAM Memory Write-Back */
 	/* To prevent data confusion issue in PSRAM */
 	mpu_entry = mpu_entry_alloc();
-	mpu_cfg.region_base = 0x02000000;
-	mpu_cfg.region_size = 0x400000;
+	/* The index of kregionx_start and kregionx_size should be matched with
+	 * PSRAM index in CONFIG_RAM_KREGIONx_START and CONFIG_RAM_KREGIONx_SIZE.
+	 */
+	mpu_cfg.region_base = (uint32_t)kregionx_start[1];
+	mpu_cfg.region_size = kregionx_size[1];
```

Comment:
According to previous code, the WB cache setting applied to whole 4MB psram. 
But according to this change, it applies only to kernel heap part. 
For the TZ area of PSRAM, the WB setting is missed. Is this ok?

Reply:
For loadable case, PSRAM_S region will not be set to wb.
We should discuss more how to.
1) change the sequence for kernel and user order.
2) add one more mpu region for PSRAM_S.
After discussing, I will update on another PR.
Thanks.

Reply:
Okay